### PR TITLE
fix float value being attempted to used as array

### PR DESCRIPTION
**Pull request by sserbin**

### CODE REVIEW

File: Classes/PHPExcel/Cell/DefaultValueBinder.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ public static function dataTypeForValue($pValue = null) {
             return PHPExcel_Cell_DataType::TYPE_STRING;
         } elseif ($pValue instanceof PHPExcel_RichText) {
             return PHPExcel_Cell_DataType::TYPE_INLINE;
-        } elseif ($pValue[0] === '=' && strlen($pValue) > 1) {
+        } elseif (!is_float($pValue) && $pValue[0] === '=' && strlen($pValue) > 1) {
```
Comment (pwnb0t) [inline Classes/PHPExcel/Cell/DefaultValueBinder.php:90]: not sure I understand what this line is saying. If `$pValue` is an array, then what happens with `strlen($pValue)`?

Comment (sserbin) [reply Classes/PHPExcel/Cell/DefaultValueBinder.php:90]: > If $pValue is an array

I'm pretty sure this clause handles `$pValue` being a string (that starts with `=`, which `[0]` checks for)

Comment (sserbin) [reply Classes/PHPExcel/Cell/DefaultValueBinder.php:90]: changed this to `is_string()` guard now, as per discussion with @kip9

**Comment by sserbin** [comment]
will re-open as GH doesn't include my last commit here (from today) :shrug:

**Comment by sserbin** [comment]
new pr - https://github.com/formstack/PHPExcel/pull/5

**Pull request closed without merging**